nixos/modules/programs/rust-motd.nix

```diff
@@ -7,7 +7,7 @@ let
   format = pkgs.formats.toml { };
 in {
   options.programs.rust-motd = {
-    enable = mkEnableOption "rust-motd";
+    enable = mkEnableOption (lib.mdDoc "rust-motd");
```

Comment:
Not sure about this (and future `mkEnableOption` calls): Is this correct?
Because looking at the implementation of `mkEnableOption`, it seems like that would try to embed an attrset into a string.

Reply:
To be more clear: I think this would result in:
```
lib.mkOption {
  type = lib.types.bool;
  default = false;
  example = true;
  description = "Whether to enable { _type = "mdDoc"; text = "rust-motd"; }.";
};
```